via/views/proxy.py

```diff
@@ -8,7 +9,8 @@
 @view_config(route_name="proxy", renderer="via:templates/proxy.html.jinja2")
 def proxy(request):
     # Strip leading '/' and normalize URL.
-    url = url_from_user_input(request.path[1:])
+    url = url_from_user_input(request.path_qs[1:])
+    url = Configuration.strip_from_url(url)
```

Comment:
Hmm. I wonder if this call belongs in `url_from_user_input()`

Reply:
Yep I've moved that in now